mllib/src/main/scala/org/apache/spark/ml/classification/LinearSVC.scala

```diff
@@ -313,7 +313,8 @@ class LinearSVCModel private[classification] (
   setDefault(threshold, 0.0)
 
   @Since("2.2.0")
-  def setWeightCol(value: Double): this.type = set(threshold, value)
+  @deprecated("This method is deprecated and will be removed in the future.", "2.4.4")
```

Comment:
Since we already removed this, shall we use `in 3.0.0` instead of `in the future`?

Reply:
I am not sure but `2.4.5` maybe anothor option?

Reply:
Nope. That's radical. We don't remove API in `maintenance` release. `3.0.0` is natural for this kind of removal.